src/Controls/src/Core/View.cs

```diff
@@ -83,6 +84,7 @@ static void OnMarginBottomPropertyChanged(BindableObject bindable, object oldVal
 
 		protected internal View()
 		{
+			_gestureManager = new GestureManager(this);
```

Comment:
This also happens in OnHandlerChangedCore and OnHandlerChangingCore in the View.Impl.cs. Does that matter?

Reply:
does it? I'm not finding the code you're referring to

![image](https://user-images.githubusercontent.com/5375137/213338476-34c589a8-8de8-4ce9-aab0-a53af6a12a49.png)

Reply:
No idea what I was looking at...